### PR TITLE
Firebase env

### PR DESCRIPTION
**Pull request by CuriouzK0d3r**
Docs and envars for firebase PR

### Description of PR

What component(s) does this PR affect?

- [ ] Back-End (Database, Detection/Configuration/etc. Microservices)
- [x] Front-End (Flask, API, UI, etc)
- [ ] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [x] Docs
- [ ] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Detection/Configuration/etc. Microservices)
- [ ] Front-End (Flask, API, UI, etc)
- [ ] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [ ] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -476,7 +476,7 @@ services:
         volumes:
             - ./mongo-data:/data/db
     nginx:
-        image: nginx:1.17-alpine
+        image: nginx:1.23-alpine
```
Comment (vkotronis) [inline docker-compose.yaml:479]: these upgrades should be part of separate PR, since this concerns only firebase

File: artemis-chart/values.yaml [hunk 2]
```diff
@@ -45,6 +45,7 @@ limitRequests: 20
 captchaWindow: 900000
 captchaTries: 4
 artemisWebBaseDir: ""
+serviceAccountPath: ""
```
Comment (vkotronis) [inline artemis-chart/values.yaml:48]: shouldn't also `.env` and  the  docker-compose files where this env var is passed be updated?

**Review by vkotronis — CHANGES_REQUESTED** [review]
Please isolate the docker container upgrades from this PR and add the env  vars  also in compose where needed

File: docker-compose.yaml (hunk 1 above)
Comment (CuriouzK0d3r) [reply docker-compose.yaml:479]: > Please isolate the docker container upgrades from this PR and add the env vars also in compose where needed
Sorry! Pushed wrong docker-compose file!
